server/Dockerfile

```diff
@@ -7,6 +7,8 @@ COPY . .
 RUN pip3 install pipenv
 RUN pipenv install
 
-#CMD ["pipenv", "run", "flask", "run", "--host", "0.0.0.0"]
 
-CMD pipenv run flask db upgrade && pipenv run flask run --host=0.0.0.0
\ No newline at end of file
+CMD pipenv run flask db stamp head && \
```

Comment:
Clarification: What's the purpose of "stamp head" here?

Reply:
Here is a good discussion of the issue.  https://stackoverflow.com/questions/17768940/target-database-is-not-up-to-date

In short, it indicates that the current state of the database represents the application of all migrations.